### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by orangeowie4** [Slimefun-Addon-Community/Galactifun#87] [Ideas]
- More efficient fuel, or the higher tier rocket the more fuel efficient it is or add rocket enhancers.
- Add better versions of space suits like ones with more oxygen capacity, better material
- Add faster machines for filling oxygen tanks in suits
- Make it so you can use the oxygen filler machines on other planets inside of a sealed room with oxygen for long-term trips or make a second tier with this ability.

**Comment by Seggan** [comment]
1 and 2 are on the todo list, 4... isn't it already like that?

**Reply by orangeowie4** [reply]
Nope. I tried it on the moon and mars.

**Reply by orangeowie4** [reply]
Thought of another, maybe oxygen tanks that you can bring with you and switch them out on your chestplate?
